include/sway/server.h

```diff
@@ -48,6 +48,8 @@ struct sway_server {
 
 	list_t *transactions;
 	list_t *dirty_containers;
+
+	list_t *scratchpad; // struct sway_container
```

Comment:
I think this should be part of `struct sway_root` instead of `struct sway_server`